### PR TITLE
Update end-life-cycle-groups-teams-sites-yammer.md

### PR DESCRIPTION
**Pull request by msbemba**
Reconstructing a teams chat conversation is now possible - https://docs.microsoft.com/en-us/microsoft-365/compliance/conversation-review-sets?view=o365-worldwide

updated the information in the Teams Section

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4932

File: microsoft-365/solutions/end-life-cycle-groups-teams-sites-yammer.md [hunk 1]
```diff
@@ -74,7 +74,8 @@ Teams-specific content is primarily in the form of conversations.
 
 Conversations in channels cannot be copied or moved using native Microsoft Teams functionality. They can however be exported using the Graph API.
 
-Additionally, if a retention policy is applied to Teams, the conversations are retained and available through eDiscovery searches. (Items found in eDiscovery searches can be exported, however there no context or structure from their original source remains – they are simply individual messages.)
+Additionally, if a retention policy is applied to Teams, the conversations are retained and available through eDiscovery searches. Using advanced eDiscovery you can [reconstruct a Teams chat conversations](/microsoft-365/compliance/conversation-review-sets)
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/end-life-cycle-groups-teams-sites-yammer.md]:
```suggestion
Additionally, if a retention policy is applied to Teams, the conversations are retained and available through eDiscovery searches. Using advanced eDiscovery you can [reconstruct a Teams chat conversation](/microsoft-365/compliance/conversation-review-sets).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**